src/main/webapp/index-script.js

```diff
@@ -162,24 +161,31 @@ function handleSearchResults(results, service) {
   // Center on the queried location
   if (results.length > 0) { 
     map.setCenter(results[0].geometry.location);
+    console.log(results.length);
   }
   
   var promises = [];
-  results.forEach((result)=> {
-    var request = {
-      placeId: result.place_id,
-      fields: [
-        'name',
-        'vicinity',
-        'reviews',
-        'place_id',
-        'opening_hours',
-        'geometry',
-        'icon',
-        'international_phone_number',
-        'website'
-      ]
-    };
+
+  // Modal can handle up to 9 results only
```

Comment:
Are we deciding to only show the 9 most crucial?

Reply:
During bug bash, Erick and I found that if we have more than 9 results, the modal breaks and the modal doesn't appear at all. We will need to debug this further, but for now, it works for up to 9.